articles/zcash-protocol/chapter_3.md

```diff
@@ -0,0 +1,209 @@
+    - $pk_d :$  $\texttt{KA}^{Sapling}\texttt{.PublicPrimeSubgroup}$ is the diversifier transmission key of the recipient’s payment address.
+    - $v$ : $\{0..$MAX_MONEY$\}$ is an integer representing the value of not in $\texttt{zatoshi}$.
+    - $rcm:$ $\texttt{NoteCommit}^{Sapling}\texttt{.Trapdoor}$ is a random commitment trapdoor.
+
+$\Longrightarrow$ Let $\texttt{Note}^{Sapling}$ be the type of a Sapling note.
+
+- An **Orchard** note is a tuple of $(d, pk_d, v, \rho, \psi, rcm)$ where:
+    - $d:\mathbb{B}^{[ℓ_d]}$ is the diversifier of the recipient’s shielded payment address.
+    - $pk_d:$ $\texttt{KA}^{Orchard}\texttt{.Public}$ is the diversifier transmission key of the recipient’s shielded payment address.
+    - $v:\{0..2^{ℓ_{value}}-1\}$ is an integer representing the value of note in $\texttt{zatoshi}$.
+    - $\rho:\mathbb{F}_{q_\mathbb{P}}$ is used as input to $\texttt{PRF}^{nfOrchard}_{nk}$ as part of deriving the nullifier of the note.
+    - $\psi:\mathbb{F}_{q_\mathbb{P}}$ is additional randomness used in deriving the nullifier.
+    - $rcm:$ $\texttt{NoteCommit}^{Orchard}\texttt{.Trapdoor}$ is a random commitment trapdoor.
+
+$\Longrightarrow$ Let $\texttt{Note}^{Orchard}$ be the type of an Orchard note.
+
+### 3.2.1 Note Plaintexts and Memo Fields
+---
+A note plaintext includes a 512-byte memo field associated with this note. The usage of the memo field is by agreement between the sender and the recipient of the note.
+
+Each Sprout note plaintext (denote $\mathrm{np}$) consists of: 
+	($\texttt{leadByte}$ : $\mathbb{B}^\mathbb{Y}$, $v : \{0..2^{ℓ_{value}}-1\}$, $\rho:\mathbb{B}^{[ℓ^{Sprout}_{PRF}]}$, $rcm:$ $\texttt{NoteCommit}^{Sprout}\texttt{.Trapdoor}$, memo : $\mathbb{B}^{\mathbb{Y}^{[512]}}$).
+The field of $\texttt{leadByte}$ is always $\texttt{0x00}$ for Sprout.
+
+Each Sapling and Orchard note plaintext (denote $\mathrm{np}$) consists of:
+	($\texttt{leadByte}$: $\mathbb{B}^\mathbb{Y}$, d : $\mathbb{B}^{[ℓ_d]}$, $v : \{0..2^{ℓ_{value}}-1\}$, $\texttt{rseed}$ : $\mathbb{B}^{\mathbb{Y}^{[32]}}$, $\texttt{memo}$ : $\mathbb{B}^{\mathbb{Y}^{[512]}}$).
+The field $\texttt{leadByte}$ indicates the version of the encoding of a Sapling or an Orchard note plaintext. For Sapling it is $\texttt{0x01}$ before activation of the $\mathrm{Canopy}$ network update and $\texttt{0x02}$ afterward. For Orchard, it is always $\texttt{0x02}$.
+
+### 3.2.2 Note Commitments
+---
+When a note is created as an output of a transaction, only a commitment to the note content is disclosed publicly in the associated $\texttt{JoinSplit}$ description, Output description, or Action description. If the transaction is entered into the blockchain, each such note is appended to the note commitment tree of the associated tree state. This allows the value and recipient to be kept hidden, while the commitment is used by zk-SNARK proof when the note is spent, to check if this note exists on the blockchain.
```

Comment:
I see that JoinSplit description is formatted with \texttt, but Output and Action description aren't. Did you forget to format them?